prow/config.yaml

```diff
@@ -2373,6 +2400,26 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
 
+- interval: 6h
+  name: ci-kubernetes-e2e-kubeadm-dind-ipv6
```

Comment:
let's just add the presubmit or the CI job while we work on this. with a presubmit you can request it manually on any schedule on a dummy pr like 46662, vs this one will run every 6 hours and be broken initially until we sort it out. we shouldn't add both immediately